src/app/todo/components/todoItem/todo-item.component.html

```diff
@@ -1,17 +1,24 @@
 <div class="flex justify-center">
     <div 
     class="w-1/3 mt-4"
-    *ngIf="taskItem$ | async as taskItem$">
-    <p class="text-lg font-semibold">{{taskItem$.title}}</p>
-    <p>{{taskItem$.description}}</p>
+    *ngIf="(taskItem$ | async) as taskItem$">
```

Comment:
when async'ing you can remove the $ from the new variable name as its no longer an observable